infra/aliyun/ali_provider.go

```diff
@@ -110,6 +112,9 @@ func (a *AliProvider) ReconcileResource(resourceKey string, action Alifunc) erro
 			return err
 		}
 		logger.Info("create resource success %s: %s", resourceKey, a.Cluster.Annotations[resourceKey])
+		if err := utils.SaveClusterfile(a.Cluster); err != nil {
```

Comment:
err = utils.SaveClusterfile(a.Cluster) is enough.
and I guess it's better to unify code style within one code block.
err = utils.SaveClusterfile(a.Cluster)
if err ...
as below.